content/joysticks/winwing/winwing-cdu/troubleshooting/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Troubleshooting
+weight: 30
+---
+
+#### I do not want MobiFlight to take over the CDU
+
+Disable it in **Extras->Settings->Peripherals**.
```

Comment:
Information on how to disable should link to our existing guide on disabling joysticks, rather than just pointing to the menu item.

> To prevent MobiFlight from taking over the CDU, disable it by following the [disabling joysticks](/joysticks/disabling/) guide.

Reply:
All headings in this file should be level two not level four, for consistency with our other troubleshooting guides.

Reply:
Changed headings to level two, but I think they look huge like that.